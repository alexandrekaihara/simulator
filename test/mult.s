LOAD TEN
MULT MINUSFIVE
ADD FIFTY
JMPZ ISZERO
OUTPUT MINUSONE
STOP
ISZERO: LOAD TEN
MULT FIVE
ADD MINUSFIFTY
JMPZ ISZERO2
OUTPUT MINUSONE
STOP
ISZERO2: OUTPUT ONE
STOP
TEN: CONST 10
FIVE: CONST 5
FIFTY: CONST 50
MINUSFIFTY: CONST -50
MINUSTEN: CONST -10
MINUSFIVE: CONST -5
ONE: CONST 1
MINUSONE:CONST -1
