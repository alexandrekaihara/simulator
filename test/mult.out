10 25 3 30 1 27 8 11 13 32 14 10 25 3 26 1 28 8 22 13 32 14 13 31 14 10 5 50 -50 -10 -5 1 -1